### PR TITLE
New edits

### PR DESCRIPTION
**Pull request by e-marshall**
- fixed errors related to dataset updates
- update to rgi7
- rewrote a few sections
- added .py file of tools

### CODE REVIEW

File: glacier_analysis_grouped.ipynb [hunk 1]
```diff
@@ -34,14 +34,14 @@
     "import pandas as pd\n",
```
Comment (JessicaS11) [inline glacier_analysis_grouped.ipynb:34]: <pre class="ql-syntax" spellcheck="false">Line #2.    se_asia = gpd.read_file('/home/emmamarshall/Desktop/data/rgi/south_asia_east_15/15_rgi60_SouthAsiaEast.shp')
</pre><p>consider including a parquet file here too (as was done in a previous notebook)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/e-marshall/itslive/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (JessicaS11) [inline glacier_analysis_grouped.ipynb:34]: <p>project --&gt; reproject; netcdf --&gt; raster</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/e-marshall/itslive/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (JessicaS11) [inline glacier_analysis_grouped.ipynb:34]: <p>project --&gt; reproject; itslive --&gt; ITS_LIVE</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/e-marshall/itslive/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (JessicaS11) [inline glacier_analysis_grouped.ipynb:34]: <p>usind --&gt; using</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/e-marshall/itslive/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

**Comment by JessicaS11** [comment]
This is looking great, @e-marshall! I added a bunch of (minor) notes via ReviewNB, and also wanted to comment that the summary and acknowledgements pages of the book are very similar. I added comments to this PR because it looked to be the current working version of the tutorials (and I could comment on individual notebook cells) - please redirect me if that's not the case!

**Comment by e-marshall** [comment]
Thanks so much for your edits @JessicaS11! and apologies that I must have never merged the PR from Nov, thanks for catching that! I'll incorporate these notes and work on distinguishing the summary and acknowledgments

**Pull request merged**